### ISSUE TITLE
"libLLVM-10.so" not found on rustc-nightly 1.47

### ISSUE THREAD

**Issue by muyopongo** [termux/termux-packages#6180]
**Problem description**

error when cargo compiling 
```
CANNOT LINK EXECUTABLE "rustc": library "libLLVM-10.so" not found
```

command
```
cargo build --target wasm32-unknown-unknown
```

rustc-nightly 1.47 requires llvm10.

https://github.com/rust-lang/rust/issues/71650

**Additional information**

```
CPU architecture
aarch64

Subscribed repositories:

sources.list
deb https://dl.bintray.com/termux/termux-packages-24/ stable main

sources.list.d/unstable.list
deb https://dl.bintray.com/xeffyr/unstable-packages unstable main

sources.list.d/x11.list
deb https://dl.bintray.com/xeffyr/x11-packages x11 main
```
:(

**Comment by Grimler91** [comment]
Yes, it is not updated after the llvm update yet. Build fails in CI: https://github.com/termux/unstable-packages/pull/208

**Comment by buttaface** [comment]
Fixed by termux/unstable-packages#209?

**Comment by muyopongo** [comment]
thank you.
